src/README.md

```
@@ -106,5 +120,21 @@
+
+
+- [Cloud Posse Documentation](https://docs.cloudposse.com) - Complete documentation for the Cloud Posse solution
+
+- [Reference Architectures](https://cloudposse.com/) - Launch effortlessly with our turnkey reference architectures, built either by your team or ours.
+
+- [Spacelift IDP Group Mapping Documentation](https://docs.spacelift.io/concepts/user-management/admin.html#idp-group-mapping) - Official Spacelift documentation for IDP group mapping
+
+
+
+
+[<img src="https://cloudposse.com/logo-300x69.svg" height="32" align="right"/>](https://cpco.io/homepage?utm_source=github&utm_medium=readme&utm_campaign=cloudposse-terraform-components/aws-spacelift-idp-group-mappings&utm_content=)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)